src/frontend/src/lib/types/utils.ts

```diff
@@ -1 +1,3 @@
 export type RequiredExcept<T, K extends keyof T> = Required<Omit<T, K>> & Pick<T, K>;
+
+export type SuccessOrNot<T = unknown> = { success: boolean; err?: T };
```

Comment:
There are few types that were defined on purpose with no `err`. Can we have two types?

For the types if `err?: unknown`, in another PR, maybe thinking that we can improve the type like `{success: true} | {success: false, err: unknown}` or something?

Reply:
Nvm my comment.